pkg/limayaml/limayaml.go

```diff
@@ -66,6 +66,7 @@ const (
 	QEMU VMType = "qemu"
 	VZ   VMType = "vz"
 	WSL2 VMType = "wsl2"
+	EXT  VMType = "ext"
```

Comment:
Maybe this should be called "external" to avoid confusion with "extended"

Reply:
In minikube we ended up calling the driver "ssh", but I don't think it's great

Reply:
ping @lima-vm/maintainers @lima-vm/reviewers RFC

I still feel "ext" is confusing.
Sounds like some sort of ext2/ext3/ext4 stuff.

Reply:
I think `external` is fine. `unmanaged` or `raw` could be alternatives, but I prefer `external`.

Reply:
ping @afbjorklund WDYT?